caproto/threading/client.py

```diff
@@ -1073,6 +1093,12 @@ def _process_search_results_loop(self):
             # and tracking circuit state, as well as a ClientChannel for
             # tracking channel state.
             for name in names:
+                log = logging.LoggerAdapter(search_logger, {'pv': name,
```

Comment:
Same issue again here---no point in using `LoggerAdapter` if it's only going to be used once.